plugins/catalog/src/components/EntitySwitch/conditions.test.ts

```diff
@@ -59,6 +78,26 @@ const bNamespace: Entity = {
   spec: { type: 'service' },
 };
 
+describe('isResourceType', () => {
+  it('should false on non component kinds', () => {
+    const checkEntity = isResourceType('service');
+
+    expect(checkEntity(notComponent)).not.toBeTruthy();
```

Comment:
notComponent wasn't really a variable i expected here :) this reads odd overall

Reply:
Thankyou for feedback - addressed in new commit